lib/modules/modules.go

```diff
@@ -383,7 +383,9 @@ func (p *defaultModules) IsEnterpriseBuild() bool {
 
 // IsOSSBuild returns true for [defaultModules].
 func (p *defaultModules) IsOSSBuild() bool {
-	return true
```

Comment:
It looks like this is intended to return true for both OSS and community editions. Can we use `BuildType` to differentiate these two editions instead?

Reply:
just wanted ensure that by adding @avatus in review

Reply:
I think this method name is confusing as it technically should be true if BuildType is "oss" or "community". This returns `false` in enterprise builds because we use `e/modules` instead. Really, this should just be the inverse of `IsEnterpriseBuild` for whatever modules package is loaded.

I think we need to keep this as always `true`